application/lib/utils/sign_box.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import '../utils/colors.dart';
+
+class SignBox extends StatelessWidget {
+  final Widget fieldIcon;
+  final String fieldText;
+
+  const SignBox(this.fieldIcon, this.fieldText);
```

Comment:
Ez nekem egy globálisabb input mezőnek tűnik, nem olyannak, mint amit csak a bejelentkezés képernyőn lehet felhasználni. Pl, amikor valaki eseményt hoz létre, ezt lehet használni az esemény nevének bekérésekor is. Emiatt a SignBox osztálynév helyett egy általánosabb InputBox osztálynév hosszú távon jobb lehet, így újrafelhasználható lesz.